### ISSUE TITLE
CI: parallel sys tests: teardown: registry socket still seems open

### ISSUE THREAD

**Issue by edsantiago** [containers/podman#23677]
One-off (so far) in [rawhide root](https://api.cirrus-ci.com/v1/artifact/task/5197615380299776/html/sys-podman-rawhide-root-host-sqlite.log.html#t--00305):
```
[+0988s] not ok 305 teardown_suite
...
# #/vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
# #| FAIL: Socket still seems open
# #\^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by Luap99** [comment]
Is this still an issue?

**Comment by edsantiago** [comment]
Haven't seen it again. This may have been fixed in #23989, I'll close on that assumption.
